checker/checker.go

```diff
@@ -118,7 +118,13 @@ func (c *Checker) Init() (err error) {
 			User:     instance.cfg.From.User,
 			Password: instance.cfg.From.Password,
 		}
-
+		if len(instance.sourceDBinfo.Password) > 0 {
+			pswd, err2 := utils.Decrypt(instance.sourceDBinfo.Password)
+			if err2 != nil {
+				return errors.Annotatef(err2, "can not decrypt password %s", instance.sourceDBinfo.Password)
```

Comment:
maybe we need to do a trick for one of:

1. a new error rather than this `err` (returned by `utils.Decrypt`) 
2. modify `utils.Decrypt` itself 
3. modfiy `errors.Cause(err).Error()` in _dm/master/server.go_ L1721. :sweat:

currently, if check task fails when starting a task, the returned message to dmctl will be something like `illegal base64 data at input byte` because we returned the original error message, but maybe `invalid encrypted password` is better.

Reply:
I think `utils.Decrypt` should at least be modified to return the annotated error, as this `"can not decrypt password"` message has been repeated too many times in this PR

Reply:
we can avoid to contain duplicate information